lms/djangoapps/completion/tests/test_models.py

```diff
@@ -0,0 +1,100 @@
+"""
+Test models, managers, and validators.
+"""
+
+from django.core.exceptions import ValidationError
+from django.test import TestCase
+from opaque_keys.edx.keys import UsageKey
+
+from student.tests.factories import UserFactory
+
+from .. import models
+
+
+class ValidatorTestCase(TestCase):
+    """
+    Test that validate_ratio only allows floats (and ints) between 0.0 and 1.0.
```

Comment:
I thought we agreed that we would use "percent" rather than "ratio" in the spec?  Is there a strong case for the need for inconsistency?

One additional argument for "percent", which I know you don't love, is that it is precisely the fact that this ratio represents a percent that provides the rules for what are valid and invalid ratios.